### ISSUE TITLE
[Feature request] Implement configurable pin functions (LED_URX, LED_UTX, LED_I2C) on MCP2221

### ISSUE THREAD

**Issue by fijam** [adafruit/Adafruit_Blinka#507]
MCP2221 datasheet mentions alternate pin functions in the pin mux module such as LED_URX, LED_UTX and LED_I2C which can be used for visual indication of UART/I2C traffic.

![image](https://user-images.githubusercontent.com/75824/133042192-e4368e02-9c5d-44bd-93b4-5b78ae648743.png)

These are available on GP0, GP1 and GP3. It would be nice to be able to take advantage of those,

![image](https://user-images.githubusercontent.com/75824/133042302-60ed5bb4-2a7b-43b0-aefd-e16089fe0748.png)

**Comment by fijam** [comment]
I dug around a bit and figured out that alternative functions can be enabled manually. For example, for LED_I2C on GP3:

```    
from adafruit_blinka.microcontroller.mcp2221 import mcp2221
mcp2221.mcp2221.gp_set_mode(3, 0b001)
```

**Comment by caternuson** [comment]
I'm not sure how we could expose this in the context of Blinka. @makermelissa any ideas?

**Comment by makermelissa** [comment]
Yeah, sort of for i2c at least. The way I added the Pico is I added all of the pin combos in the microcontroller file. For the mcp2221, with the i2c implementation, you could figure out if it's an alternate i2c port and if so, just set the mode in the microcontroller specific implementation of i2c.
